virtual_ssd/ssd.py

```diff
@@ -91,7 +88,17 @@ def __erase_nand(self, addr: int, size: int):
         return SSD.SUCCESS
 
     def read(self, addr: int):
-        return self.__read_nand(addr)
+        if not self.__isvalid_address(addr):
+            return SSD.FAIL
+
+        read_result = self.__buffer.get_value(addr)
+
+        if read_result is not None:
+            self.__write_result_file(read_result)
+            return SSD.SUCCESS
+
+        else:
+            return self.__read_nand(addr)
```

Comment:
간단하고 좋네요.
여기에 Guard Clause 사용 어떠실까요?
```suggestion
        if read_result is None:
            return self.__read_nand(addr)
            
        self.__write_result_file(read_result)
        return SSD.SUCCESS
```

Reply:
좋은것 같습니다.
바로 반영할게요